recipes/pudb/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set version = "2016.2" %}
 
-ackage:
+package:
```

Comment:
It sort of does that. 😆

But yeah we can improve on this. Opened this issue ( https://github.com/conda-forge/conda-smithy/issues/231 ).